### PR TITLE
Split ICE candidate queue.

### PR DESCRIPTION
**Pull request by boks1971**
Shared ICE candidate queue meant only one of PUBLISHER/SUBSCRIBER pc got final candidate notification. Split the queue.

### CODE REVIEW

File: pkg/sfu/utils/wraparound.go [hunk 1]
```diff
@@ -211,6 +211,6 @@ func (w *WrapAround[T, ET]) isWrapBack(earlier T, later T) bool {
 
 // ------------------------------------
 
-func getExtendedHighest[T number, ET extendedNumber](cycles ET, val T) ET {
+func getExtended[T number, ET extendedNumber](cycles ET, val T) ET {
```
Comment (boks1971) [inline pkg/sfu/utils/wraparound.go:214]: drive-by renaming. The `Highest` does not make sense here. It is just calculating extended.

**Review by davidzhao — APPROVED** [review]
